### ISSUE TITLE
Site editor: shift tab from the first block, or tab from the last block not working as expected

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#41811]
### Description

I noticed that shift-tab (used to access the block toolbar) and tab (used to access the block inspector) aren't working as expected in the site editor. A JS error is being thrown:
```console
Uncaught TypeError: Cannot destructure property 'tagName' of 'element' as it is undefined.
    at isFormElement (is-form-element.js:15:10)
    at HTMLBodyElement.onKeyDown (use-tab-nav.js:121:18)
```

Things seem to work ok in the post editor.

### Shift-tabbing
1. Select the first block (usually a header template part, selecting via list view makes this easier)
2. Press shift tab

Expected - the toolbar is focused
Actual - nothing happens visually, a JS error is thrown

### Tabbing
1. Add an image block as the last block and select it
2. Tab through the buttons in the placeholder and keep pressing tab

Expected - the block inspector is focused
Actual - nothing happens visually, a JS error is thrown

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
